lib/resources/FileUploads.js

```diff
@@ -9,14 +10,38 @@ module.exports = StripeResource.extend({
 
   overrideHost: 'uploads.stripe.com',
 
-  requestDataProcessor: function(method, data, headers) {
+  requestDataProcessor: function(method, data, headers, callback) {
     data = data || {};
 
     if (method === 'POST') {
-      return multipartDataGenerator(method, data, headers);
+      return getProcessorForSourceType(data);
     } else {
       return utils.stringifyRequestData(data);
```

Comment:
This needs to make use of the callback flow here, so it's probably `return callback(null, utils.stringifyRequestData(data));`.

Reply:
oh wow, great catch - my fault - I totally missed that! Thanks